app/views/cities/index.html.haml

```diff
@@ -1,5 +1,14 @@
-%h1 Choose Your Adventure
+%h1.center-align Choose Your Adventure
 
-%div
+.container
   - @cities.each do |city|
-    = link_to "#{city.name}", city_path(city)
+    .row
+      %div{class: "col s8 offset-s2"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names